cmd/state-installer/test/integration/installer_int_test.go

```diff
@@ -284,7 +303,12 @@ func (suite *InstallerIntegrationTestSuite) AssertConfig(ts *e2e.Session) {
 			fname = ".zshrc"
 		}
 
-		bashContents := fileutils.ReadFileUnsafe(filepath.Join(homeDir, fname))
+		rcFile := filepath.Join(homeDir, fname)
+		if fileutils.FileExists(filepath.Join(ts.Dirs.HomeDir, filepath.Base(fname))) {
```

Comment:
Could we just test `fname` directly (instead of `filepath.Base(fname)`) and remove all usage of `homeDir` from line 295? I don't think we want to have any mention of the real home dir for fear of confusion or accidental contamination.

Reply:
I decided to go the other direction with this and unify the `AssertConfig` functions to rely on the subshell mechanic. That way the integration tests aren't assuming what shell they are running on. If you would still prefer we go the other way let me know.